Assets/_BForBoss/_Core/Scripts/Enemies/EnemyShootingBehaviour.cs

```diff
@@ -0,0 +1,116 @@
+using System;
+using Perigon.Utility;
+using Perigon.Weapons;
+using UnityEngine;
+
+namespace BForBoss
+{
+    public class EnemyShootingBehaviour : MonoBehaviour
+    {
+        [SerializeField] private float _distanceToShootAt = 4;
```

Comment:
hmm, could you add a comment (or a tooltip), describing what units this should be in? not sure what 4 is supposed to represent just by the variable name

Reply:
It's Unity units. it's arbitrary to whatever a Unity unit is.

Reply:
<img width="548" alt="Screen Shot 2022-07-20 at 7 42 34 AM" src="https://user-images.githubusercontent.com/5779541/179973697-d1caba09-6509-4bda-a38b-eb47c5cfff87.png">